libmscore/mscore.cpp

```diff
@@ -164,6 +164,7 @@ void Direction::fillComboBox(QComboBox* cb)
       }
 
 static Spatium doubleToSpatium(double d)       { return Spatium(d); }
+static Direction intToDirection(int i)         { return Direction(i); }
```

Comment:
I could add a Q_ASSERT(v.canConvert<T>()) every time we use it if you want.
During further debugging this would give us a hint, where it's not working.